lib/network/modules/NodesHandler.js

```diff
@@ -1,9 +1,8 @@
 let util = require("../../util");
 let DataSet = require('../../DataSet');
 let DataView = require('../../DataView');
-
 var Node = require("./components/Node").default;
-var Label = require("./components/shared/Label").default;
+
```

Comment:
extra line

Reply:
You mean that there are two empty lines? I actually prefer this, for better visual separation. I do it between methods as well, and in other places where I think a bigger distinction between blocks is useful.